tasks/install_wp_client.yml

```diff
@@ -0,0 +1,28 @@
+---
+# The following tasks are responsible for installing wpi-cli, the command line
+# interface for WordPress.
+
+- name: Check if wp-cli is present
+  stat:
+    path: /usr/local/bin/wp
+  register: wp_cli_exec
+
+- name: Download wp-cli
+  get_url:
+    url: 'https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar'
+    dest: "{{ global_cache_dir | mandatory }}/"
+  become: false
+  delegate_to: localhost
+  when: not wp_cli_exec.stat.exists
+
+- name: Install wp-cli
+  copy: 
+    src: "{{ global_cache_dir }}/wp-cli.phar"
+    dest: "/usr/local/bin/wp"
+    mode: 0755
+    owner: root
+    group: root
+  when: not wp_cli_exec.stat.exists
+
+- name: Update wp-cli
+  shell: wp cli update
```

Comment:
As root?

Reply:
Yes. Root is owner of the command line interface for WordPress (wp-cli) that makes it possible to deploy WordPress instances from the shell. The WordPress instances themself belong to www-data